apps/desktop/src/lib/hunk/HunkDiff.svelte

```diff
@@ -485,14 +500,36 @@
 			transform 0.2s;
 	}
 
+	.table__right-box {
+		pointer-events: none;
+		position: absolute;
```

Comment:
> I don't really like this approach

That's fair, and if you feel strongly against it we can dismiss this approach completely.

I like it because it can look like the designs, even-though the price is this complicated structure of elements dynamically positioned elements.

> Another issue with this approach is shifting when you zoom the app. It's because there are px untis instead of rem
>

You're right! Using pxToRem seems to have solved it for me, though

https://github.com/user-attachments/assets/eb35fc2b-d268-4816-bc12-d980fc8c37bb